### PR TITLE
Add aiohttp subscription support

### PR DESCRIPTION
**Pull request by DoctorJohn**
## Description

This PR adds subscription support to the aiohttp integration. Note that aiohttp does not support ASGI and thus can't reuse the existing subscription implementation. This PR also includes various tests (100% test coverage). Some of them could be reused (in a future PR) to possibly increase the ASGI implementations test coverage.

Both subscription implementations obviously share many similarities. I will do some experiments to explore how to abstract both implementations. (This will happen in a separate PR).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [x] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/),so make sure to pick the appropriate type. If in doubt feel free to ask :)

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        for i in range(target):
+            yield i
+            await asyncio.sleep(0.5)
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def _unused(self) -> None:
+        pass
```
Comment (patrick91) [inline RELEASE.md:30]:
```suggestion
    def _unused(self) -> str:
        return ""
```

File: strawberry/asgi/__init__.py [hunk 2]
```diff
@@ -82,7 +83,7 @@ async def handle_websocket(self, scope: Scope, receive: Receive, send: Send):
         subscriptions: typing.Dict[str, typing.AsyncGenerator] = {}
         tasks = {}
 
-        await websocket.accept(subprotocol="graphql-ws")
+        await websocket.accept(subprotocol=GRAPHQL_WS)
```
Comment (patrick91) [inline strawberry/asgi/__init__.py:86]: 💯

**Review by patrick91 — APPROVED** [review]
This looks awesome! Nice one! 👏

**Comment by patrick91** [comment]
GitHub actions are taking a long time today 🤔

**Comment by DoctorJohn** [comment]
Apparently there's an incident with github actions. https://www.githubstatus.com/

**Pull request merged**